stories/Autosuggest.js

```diff
@@ -75,7 +75,8 @@ storiesOf('Organisms|Autosuggest', module)
         };
 
         const getSelectedPlaceholder = () => {
-            const numOfItems = store.get('selectedSuggestions').length;
+            const items = store.get('selectedSuggestions');
+            const numOfItems = items ? items.length : 0;
```

Comment:
Matter of 'taste' but I think this could be even simpler like this:

`const numOfItems = items.length || 0`

Either way it fixes the issue.

Reply:
if we use this construct you would get the same error as before, when `items` is undefined

Reply:
Ah true